coreneuron/utils/tokenizer.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+std::vector<std::string> tokenize ( const char* str,
```

Comment:
format is very simple: replace this with split or use of strtok

Reply:
initially we were supporting more,probably I keep it like that for now on as it can change.

Reply:
add this function for parsing ***channelName.varName*** is overkill :)